README.md

```diff
@@ -150,26 +150,26 @@ income_general               income
  Account                       Balance             
 ---------------------------------------
 asset
-  checking                     1,300               
-  savings                      2,000               
+  checking                     1300.00               
+  savings                      2000.00               
 liability
-  credit_card                  -456                
+  credit_card                  -456.00                
 equity
-  equity                       -3,500              
+  equity                       -3500.00              
 expense
-  expense_auto                 455                 
-  expense_computer             1                   
+  expense_auto                 455.00                 
+  expense_computer             1.00                   
   expense_food                 0                   
   expense_gasoline             0                   
   expense_pets                 0                   
   expense_amazon               0                   
-  expense_home                 100                 
+  expense_home                 100.00                 
   expense_general              0                   
 income
   income_general               0                   
 
 ---------------------------------------
-check                          -100 
+check                           -100.00 
```

Comment:
could we adjust the example numbers (maybe add an expense of 100) to ensure that the `check` is `0`? We'd have to the individual transaction to the `register` example for consistency, too. Even though the `check` is a feature, I think it might cause confusion if the ledger was out of balance for simple examples.

Reply:
Sure can. I will make this update and refresh all the examples with fresh data.